x-pack/plugins/security_solution/public/resolver/view/panels/use_formatted_date.test.tsx

```diff
@@ -4,103 +4,53 @@
-      const formattedDate = useFormattedDate(date);
-      return <div data-test-subj={testID}>{formattedDate}</div>;
+      return <>{useFormattedDate(date)}</>;
     }
 
-    reactRenderResult = (
-      date: ConstructorParameters<typeof Date>[0] | Date | undefined
-    ): RenderResult =>
-      render(
+    formattedDate = (date: ConstructorParameters<typeof Date>[0] | Date | undefined): string =>
+      mount(
         <KibanaContextProvider services={mockCoreStart}>
           <Test date={date} />
         </KibanaContextProvider>
-      );
-  });
-  afterEach(() => {
-    jest.clearAllMocks();
-  });
-
-  describe('when the provided date is undefined', () => {
-    it('should return undefined', async () => {
-      const { findByTestId } = reactRenderResult(undefined);
-      element = await findByTestId(testID);
-
-      expect(element).toBeEmptyDOMElement();
-    });
-  });
-
-  describe('when the provided date is empty', () => {
-    it('should return undefined', async () => {
-      const { findByTestId } = reactRenderResult('');
-      element = await findByTestId(testID);
-
-      expect(element).toBeEmptyDOMElement();
-    });
-  });
-
-  describe('when the provided date is an invalid date', () => {
-    it('should return the string invalid date', async () => {
-      const { findByTestId } = reactRenderResult('randomString');
-      element = await findByTestId(testID);
-
-      expect(element).toHaveTextContent('Invalid Date');
-    });
-  });
-
-  describe('when the provided date is a stringified unix timestamp', () => {
-    it('should return the string invalid date', async () => {
-      const { findByTestId } = reactRenderResult('1600863932316');
-      element = await findByTestId(testID);
-
-      expect(element).toHaveTextContent('Invalid Date');
-    });
-  });
-
-  describe('when the provided date is a valid numerical timestamp', () => {
-    it('should return the string invalid date', async () => {
-      const { findByTestId } = reactRenderResult(1600863932316);
-      element = await findByTestId(testID);
-
-      expect(element).toHaveTextContent('Sep 23, 2020 @ 08:25:32.316');
-    });
-  });
-
-  describe('when the provided date is a date string', () => {
-    it('should return the string invalid date', async () => {
-      const { findByTestId } = reactRenderResult('2020-09-23T12:25:32Z');
-      element = await findByTestId(testID);
-
-      expect(element).toHaveTextContent('Sep 23, 2020 @ 08:25:32.000');
-    });
-  });
-
-  describe('when the provided date is a valid date', () => {
-    it('should return the string invalid date', async () => {
-      const validDate = new Date(1600863932316);
-      const { findByTestId } = reactRenderResult(validDate);
-      element = await findByTestId(testID);
-
-      expect(element).toHaveTextContent('Sep 23, 2020 @ 08:25:32.316');
-    });
+      ).text();
+  });
+
+  it.each([
```

Comment:
These look like this now:
![image](https://user-images.githubusercontent.com/35559/96640929-eeef2380-12f1-11eb-8181-5af4f945f7e3.png)